docker-compose.regtest.yml

```diff
@@ -1,4 +1,22 @@
 services:
```

Comment:
please revert the changes, but let's not forget to make use of inmemory live store, badger event repo and sqlite repo

Reply:
Done